frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -60,6 +60,25 @@ function hasVote(
   );
 }
 
+function hasWriteInAtPosition(
+  vote: CandidateVote | undefined,
+  position: number
+): boolean {
+  return (
+    vote?.some((candidate) => candidate.writeInPosition === position) ?? false
+  );
+}
+
+function writeInNameAtPosition(
```

Comment:
Since `writeIn` can also be read as a verb, what do you think of renaming this `getWriteInNameAtPosition` to make it clear that we're just retrieving a value and not actually writing in a name?